cg/meta/upload/scoutapi.py

```diff
@@ -49,16 +48,19 @@ def build_samples(self, analysis_obj: models.Analysis, hk_version_id: int = None
             sample_id = link_obj.sample.internal_id
             bam_path = self.fetch_file_path("bam", sample_id, hk_version_id)
             alignment_file_path = self.fetch_file_path("cram", sample_id, hk_version_id)
-            chromograph_path = self.fetch_file_path("chromograph", sample_id, hk_version_id)
+            chromograph_path = self.fetch_file_path(
+                "chromograph", sample_id, hk_version_id
+            )
             mt_bam_path = self.fetch_file_path("bam-mt", sample_id, hk_version_id)
-            vcf2cytosure_path = self.fetch_file_path("vcf2cytosure", sample_id, hk_version_id)
+            vcf2cytosure_path = self.fetch_file_path(
+                "vcf2cytosure", sample_id, hk_version_id
+            )
 
             lims_sample = dict()
             try:
-                lims_sample = self.lims.sample(sample_id)
+                lims_sample = self.lims.sample(sample_id) or {}
```

Comment:
It is instantiated as a empty dict above. Code below will expect a dict. The function below the `try:` can return `None` and it that case the code below would break when treating it as a dict